bedrock/firefox/templates/firefox/whatsnew/whatsnew-base-42.html

```diff
@@ -72,6 +72,11 @@ <h1 data-alt="{{ _('The world’s most private browsing. Only from Firefox.') }}
 
 {% block email_form %}{% endblock%}
 
-{% block js %}{% endblock %}
+{% block js %}
+  {% if switch('statcounter-experiment') %}
+    {% javascript 'stat_counter' %}
+    {% include 'includes/stat_counter.html' %}
+  {% endif %}
+{% endblock %}
```

Comment:
No need to update this template, as it is only used as a base for `whatsnew-42.html` and `whatsnew-zh-tw-49.html` (which are updated in this PR to contain the StatCounter stuff).